README.md.gotmpl

```diff
@@ -1,16 +1,67 @@
-{{ template "chart.header" . }}
+# Woodpecker Helm Charts
```

Comment:
Please adjust the `README-main.md.gotmpl` for most changes as this one is used by the individual charts.